script.js

```diff
@@ -1,4 +1,309 @@
+//Add global states to include wins and draws
+var playerScore = 0;
+var comScore = 0;
+var drawScore = 0;
+
+//Add states for username and game mode
+var userName = "";
+var gameMode = "";
+
+//Game mode checker
+var gameModeState = 0;
+
+//Add WinState for korean mode
+var playerLastWinState = 0;
+var comLastWinState = 0;
+var winnerState = 0;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  //Convert input to lowercase and remove white space, allow for minor typos
+  var convertInput = input.toLowerCase().replace(/\s/g, "");
+
+  //Prints error message if input is not SPS or reversed SPS
+  var myOutputValue = `Please enter 'stone', 'paper' or 'scissors' <br>Current Score<br>User:${playerScore}<br>Com:${comScore}<br>Draws:${drawScore}`;
+
+  //If userName is not set, use first input as username
+  if (userName == "") {
+    if (input == "") {
+      return `Please input something for your username`;
+    }
+
+    userName = input;
+
+    return `Hi ${userName}, Please enter game mode to start playing.`;
+  }
```

Comment:
Thanks for the comment above this chunk of code.
Like that you are taking steps to make it personal for the user!